src/AstUtils.h

```diff
@@ -72,6 +74,67 @@ std::vector<T*> getBodyLiterals(const C& clause) {
     return res;
 }
 
+/**
+ * Returns a vector of clauses in the program describing the relation with the given name.
+ *
+ * @param program the program
+ * @param name the name of the relation to search for
+ * @return vector of clauses describing the relation with the given name
+ */
+std::vector<AstClause*> getClauses(const AstProgram& program, const AstQualifiedName& relationName);
+
+/**
+ * Returns a vector of clauses in the program describing the given relation.
+ *
+ * @param program the program
+ * @param rel the relation to search for
+ * @return vector of clauses describing the given relation
+ */
+std::vector<AstClause*> getClauses(const AstProgram& program, const AstRelation& rel);
```

Comment:
Could we simplify this and only have the one getClauses method that uses `AstQualifiedName`?

Reply:
This case actually crops up quite a lot, and removing it would add a lot of bloat to the code. After changing the 60th one (manually) without reaching the end, I decided this is a bad idea, and so I think we should keep it for code clarity.